app/src/ui/lib/rich-text.tsx

```diff
@@ -14,6 +14,9 @@ interface IRichTextProps {
   /** The raw text to inspect for things to highlight */
   readonly text: string
 
+  /** Should URLs be rendered as clickable links. Default true. */
+  readonly links?: boolean
```

Comment:
To make the purpose of this a bit clearer, what about a name like `renderLinks`?